content/docs/concepts/enterprise_users.md

```diff
@@ -0,0 +1,109 @@
+multi-cloud/hybrid cloud environment:
+
+![K8s MultiCluster](/img/concepts/enterprise/k8s_multi_cluster.svg)
+
+How do workloads communicate independent of where they are running?
+
+It's not just a problem of cluster to cluster communication.  In the diagram below: 
+- The Red Pods need to communicate with each other
+- The Greed Pods need to communicate with each other
+- The Red Pod with the Green outline needs to communicate with both the other Red Pods *and* the other Green Pods
+- Pods in different clusters that are neither Red nor Green should be communicating with each other.
+
+![K8s MultiCluster2](/img/concepts/enterprise/k8s_multi_cluster_2.svg)
+
+
+### Problem Statement
+
+How do you enable workloads collaborating together to produce an App communicate independent of where those workloads are running?
+
+## The NSM Solution
+
+Network Service Mesh allows individual workloads, where ever they are running to connect securely to Network Service(s) that
+are independent of where they run:
+
+![NSM Solution](/img/concepts/enterprise/nsm_solution.svg)
+
+The Kubernetes CNI of your choice provides intra-cluster networking continues for every Pod.  Network Service Mesh does not require you to replace
+your CNI, nor does it interfere in any way with what you are accustomed to getting from your CNI for intra-cluster networking.
+
+NSM is architecturally independent of the Runtime Domain.  While it supports K8s, it is not limited to it:
+
+![NSM Solution2](/img/concepts/enterprise/nsm_solution_2.svg)
+
+## NSM and Traditional Service Mesh
+
+Network Service Mesh is complementary to traditional Service Meshes like Linkerd, Istio, Kuma, and Consul.
+
+### Payloads
+Traditional Service Meshes predominantly focus on L7 payloads like HTTPS.  If a workload send an HTTPS message, they only guarantee
+that the HTTPS message itself gets to the other side and the HTTPS response gets back to the workload.  In the intervening process
+the ethernet headers, ip headers, and even the TCP connection may have been stripped away and replaced.  The payload being tranported
+across the Mesh truly is the L7 HTTPS message.
+
+Network Service Mesh provides a similar service for transporting payloads that are IP Packets.
+This can be particularly effective for certain kinds of legacy workloads, like DBs, that are using bespoke protocols
+for replication over IP:
+
+![DB Replication](/img/concepts/enterprise/db_replication.svg)
+
+### Better Together
+
+A Traditional Service Mesh itself can be viewed as a Network Service.  Network Service Mesh can be used to connect a workload
+to multiple 'Service Mesh' Network Services that are not associated with the local cluster running the workload.
+
+![Traditional Service Mesh](/img/concepts/enterprise/traditional_service_mesh.svg)
+
+## Cross Company Network Services
+
+Because Network Service Mesh decouples Network Services from the underlying Runtime Domain, it is possible to workloads from multiple companies
```

Comment:
Reword, maybe "it is possible for workloads from multiple companies" instead of of "it is possible to workloads..." ?